src/collection.js

```diff
@@ -483,6 +483,16 @@ class Collection {
 
     return this
   }
+
+  /**
+   *Returns the average of all collection items
+   *
+   * @returns {Number}
+   * */
+  async avg () {
+    const sum = this.items.reduce((total, num) => { return total + num }, 0)

```

Comment:
can you please use internal `sum` call here instead of `reduce` on array?

Reply:
Good catch! Yep, please merge `master` and use `this.sum()`